WardrobeCloset/Controllers/AccessoriesController.cs

```diff
@@ -0,0 +1,132 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Data.Entity;
+using System.Linq;
+using System.Net;
+using System.Web;
+using System.Web.Mvc;
+using WardrobeCloset.Models;
+
+namespace WardrobeCloset.Controllers
+{
+    public class AccessoriesController : Controller
+    {
+        private WardrobeClosetProjectEntities db = new WardrobeClosetProjectEntities();
+
+        // GET: Accessories
+        public ActionResult Index()
+        {
+            var accessories = db.Accessories.Include(a => a.Outfit);
+            return View(accessories.ToList());
+        }
+       
+        // GET: Accessories/Details/5
+        public ActionResult Details(int? id)
+        {
+            if (id == null)
+            {
+                return new HttpStatusCodeResult(HttpStatusCode.BadRequest);
+            }
+            Accessory accessory = db.Accessories.Find(id);
+            if (accessory == null)
+            {
+                return HttpNotFound();
+            }
+            return View(accessory);
+        }
+
+        // GET: Accessories/Create
+        public ActionResult Create()
+        {
+            ViewBag.OutfitsID = new SelectList(db.Outfits, "OutfitsID", "OutfitName");
+            return View();
+        }
+
+        // POST: Accessories/Create
+        // To protect from overposting attacks, please enable the specific properties you want to bind to, for 
+        // more details see http://go.microsoft.com/fwlink/?LinkId=317598.
+        [HttpPost]
+        [ValidateAntiForgeryToken]
+        public ActionResult Create([Bind(Include = "AccessoriesID,Name,ClothingType,OutfitsID,Brand,Photo,Color,Occasion,Season,Favorite,TimesWorn,Dirty")] Accessory accessory)
+        {
+            if (ModelState.IsValid)
+            {
+                db.Accessories.Add(accessory);
+                db.SaveChanges();
+                return RedirectToAction("Index");
+            }
+
+            ViewBag.OutfitsID = new SelectList(db.Outfits, "OutfitsID", "OutfitName", accessory.OutfitsID);
+            return View(accessory);
+        }
+
+        // GET: Accessories/Edit/5
+        public ActionResult Edit(int? id)
+        {
+            if (id == null)
```

Comment:
Note how lines 67-75 here are identical to 27-35 above. You should consider extracting the common pieces into a private method so you can re-use them in both places.